File: rife_v1_remaining/programs/unittests/scripts/jruby/inheritance_parent.rb
def childTriggered(name, value)
	if name == "trigger" && value[0] == "ok"
		return true
	else
		return false
	end
end

if $element.hasSubmission("activatechild")
	$element.setOutput("trigger", "ok")
end

$element.print(<<END)
<html>
<body>
<a href="#{$element.getSubmissionQueryUrl("activatechild")}">activate child</a>
</body>
</html>
END

File: rife_v1_remaining/programs/unittests/scripts/rhino/inheritance_parent.js
if (element.hasSubmission("activatechild"))
{
	element.setOutput("trigger", "ok");
}

element.print("<html><body><a href=\""+element.getSubmissionQueryUrl("activatechild")+"\">activate child</a></body></html>");
	
function childTriggered(name, value)
{
	if (name == "trigger" &&
		value[0] == "ok")
	{
		return true;
	}
	else
	{
		return false;
	}
}
